services/email/scripts/deploy.sh

```diff
@@ -1,4 +1,24 @@
-apiVersion: apps/v1beta1
+#!/usr/bin/env bash
+
+if [ -z $1 ]; then
+    echo "Usage: $(basename $0) CONTAINER_REGISTRY" >&2
+    exit 1
+fi
+
+root=`cd $(dirname $0); cd ..; pwd`
+
+version=`cat $root/version`
+svc_name=`basename $root`
+
+container_reg=$1
+image_name="$container_reg.azurecr.io/$svc_name:$version"
+
+az acr login -n $container_reg
```

Comment:
>login [](start = 7, length = 5)

Do we need also the k8s login?

Reply:
General yes. I will address this later. https://github.com/Azure/adx-automation-services/issues/19